.nvmrc

```diff
@@ -0,0 +1 @@
+v12.3.1
```

Comment:
Can revert this now.

Reply:
I had added this seperate from the addition on master. I'll change it to match the version in master.